lol.janjic.Splitcat.yml

```
@@ -0,0 +1,67 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: lol.janjic.Splitcat
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: splitcat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: Splitcat
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - |
+        # Exit if any command fails
+        set -e
+
+        # Echo all commands for debug purposes
+        set -x
+
+
+        # No spaces in project name.
+        projectName=splitcat
+        projectId=lol.janjic.Splitcat
+        executableName=splitcat
+
+        # ------------------------------- Build Flatpak ----------------------------- #
+
+        # Extract portable Flutter build.
+        mkdir -p $projectName
+        tar -xf $projectName-linux-x64.tar.gz -C $projectName
+
+        # Copy the portable app to the Flatpak-based location.
+        cp -r $projectName /app/
+        chmod +x /app/$projectName/$executableName
+        mkdir -p /app/bin
+        ln -s /app/$projectName/$executableName /app/bin/$executableName
+
+        # Install the icon.
+        iconDir=/app/share/icons/hicolor/256x256/apps
+        mkdir -p $iconDir
+        wget https://raw.githubusercontent.com/vogonwann/splitcat/d8f95eff7dbfaf283dbfaf93b64ebbd2302d7847/linux/flutter/assets/icons/256x256/icon.png -o $iconDir/$projectId.png
+
+        # Install the desktop file.
+        desktopFileDir=/app/share/applications
+        mkdir -p $desktopFileDir
+        cp -r packaging/linux/$projectId.desktop $desktopFileDir/
+
+
+        # Install the appdata file.
+        metadataDir=/app/share/metainfo
+        mkdir -p $metadataDir
+        cp -r packaging/linux/$projectId.appdata.xml $metadataDir/
+    sources:
+      - type: file
+        url: https://github.com/vogonwann/splitcat/releases/download/v0.1.4/splitcat-linux-x64.tar.gz
+        sha256: 7d3a7f11c4244f038f670b2017848a9baa8c80cc5ddd4fa47cef3da7336a1232
```

```suggestion
      - type: file
        url: https://github.com/vogonwann/splitcat/releases/download/v0.1.4/splitcat-linux-x64.tar.gz
        dest: splitcat
        strip-components: 0
        sha256: 7d3a7f11c4244f038f670b2017848a9baa8c80cc5ddd4fa47cef3da7336a1232
```

we could avoid the `strip-components` if the tarball was properly generated.